extensions/wikia/SponsorshipDashboard/SponsorshipDashboard.body.php

```diff
@@ -47,13 +47,13 @@ public function isAllowed() {
 
 	function execute( $subpage = false ) {
 
-		global $wgSupressPageSubtitle;
+		global $wgSuppressPageSubtitle;
 
 		$wgOut = F::app()->getGlobal('wgOut');
 
 		$wgOut->setHTMLTitle( wfMsg( 'sponsorship-dashboard-default-page-title' ) );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)